lib/plug/logger.ex

```diff
@@ -41,6 +42,15 @@ defmodule Plug.Logger do
     end)
   end
 
+  # TODO: remove this once Plug supports only Elixir 1.2.
+  if function_exported?(:erlang, :monotonic_time, 0) do
+    defp current_time, do: :erlang.monotonic_time
```

Comment:
We should use `:erlang.monotonic_time(:micro_seconds)` here and remove `convert_time_unit` call.